File: src/main/java/com/enihsyou/astolfo/hotel/HotelAstolfoApplication.java
package com.enihsyou.astolfo.hotel;

import com.enihsyou.astolfo.hotel.mybatis.UserMapper;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class HotelAstolfoApplication implements CommandLineRunner {

  public static void main(String[] args) {
    SpringApplication.run(HotelAstolfoApplication.class, args);
  }

  final private UserMapper userMapper;

  public HotelAstolfoApplication(UserMapper userMapper) {
    this.userMapper = userMapper;
  }

  @Override
  public void run(String... args) throws Exception {
    System.out.println(this.userMapper.selectByUsername("dfd"));
  }
}

File: src/main/kotlin/com/enihsyou/astolfo/hotel/HotelAstolfoApplication.kt
package com.enihsyou.astolfo.hotel

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.web.servlet.ServletComponentScan


@SpringBootApplication
@ServletComponentScan
class HotelAstolfoApplication

fun main(args: Array<String>) {
    SpringApplication.run(HotelAstolfoApplication::class.java, *args)
}


